### PR TITLE
fix: Replaced a static Google Map component with a dynamic one

### LINKED ISSUE #2171: Map is not working
**Describe the bug**
Map is not working as it is very important for user to confirm the location by seeing the map above the search bar that this is the location they are searching for, but on searching the name of location nothing happens with map.It remains as it is.

**Expected behavior**
It should show the searched location on map just above the search bar.

**Screenshots**
screenshot from 2019-02-09 21-56-38
**Additional context**
I would like to work on this issue.

### PR DESCRIPTION
**Pull request by mrsaicharan1**
map-fix

Fixes #2171 

#### Short description of what this resolves:
- This replaces the static google map with an updatable one, along with a location change handler. 

#### Changes proposed in this pull request:

- Added `onLocationChangeHandler`
- Passed dynamic longitudes and latitudes

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/templates/components/widgets/forms/location-input.hbs [hunk 1]
```diff
@@ -30,7 +30,7 @@
     </div>
     <div class="column">
       {{#g-map lat=lat lng=lng zoom=zoom class='google-maps' address=combinedAddress as |context|}}
-        {{g-map-address-marker context address=combinedAddress onLocationChange=(action 'onLocationChangeHandler' lat lng)}}
+        {{g-map-address-marker context address=combinedAddress onLocationChange=(action 'onLocationChangeHandler')}}
```
Comment (kushthedude) [inline app/templates/components/widgets/forms/location-input.hbs]: @mrsaicharan1 Why you removed the lat and lag variables here

Comment (mrsaicharan1) [reply app/templates/components/widgets/forms/location-input.hbs]: restored

**Comment by mrsaicharan1** [comment]
@uds5501 @shreyanshdwivedi Please review

File: app/utils/computed-helpers.js [hunk 2]
```diff
@@ -46,9 +46,9 @@ export const computedSegmentedLink = function(property) {
 export const computedDateTimeSplit = function(property, segmentFormat) {
   return computed(property, {
     get() {
-      if (property === 'endsAt' && segmentFormat === 'date') {
-        return moment(this.get(property)).add(1, 'days').format(getFormat(segmentFormat));
-      }
+      // if (property === 'endsAt' && segmentFormat === 'date') {
+      //   return moment(this.get(property)).add(1, 'days').format(getFormat(segmentFormat));
+      // } The following line was adding one extra day to the endsAt Attribute . Commenting it to fix the issue .
```
Comment (kushthedude) [inline app/utils/computed-helpers.js]: @mrsaicharan1 Unwanted commits are appearing , Please rebase the branch or Force Push it

**Comment by mrsaicharan1** [comment]
@uds5501 @shreyanshdwivedi @CosmicCoder96 I've attached a GIF related to the changes. Please review.

**Comment by mrsaicharan1** [comment]
@CosmicCoder96 Travis seems to have a random failure

**Pull request merged**